docs/getting-started/getting-started-ubuntu-spark-net.md

```diff
@@ -0,0 +1,187 @@
+# Getting Started with Spark.NET on Ubuntu
+
+These instructions help get you started with Spark on Ubuntu 18.04
+
+## Download and Install Prerequisites
+
+- [Java 1.8](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+- [Spark 2.4.x](https://spark.apache.org/downloads.html)
+- Maven
+- [.NET Core SDK 2.1](https://dotnet.microsoft.com/download/dotnet-core/2.1)
+
+### Install Java
```

Comment:
Instead of reiterating the install steps for java, spark, netcore and maven, we can instead reference docs/building/ubuntu-instructions.md#pre-requisites and update those instructions accordingly if necessary.   This will help if we ever need to update these instructions in the future.

Reply:
Updated. I added a link to the [home page](https://github.com/dotnet/spark#get-started) for the pre-requisites. One thing to note though is maven is not required to create an app. This is just in the event a user wants to build from source. Otherwise, they can just take the [Microsoft.Spark.Worker release package](https://github.com/dotnet/spark/releases) which seems like the easiest way to go about it.

Reply:
Another approach would be to have another document showing how to set up pre-requisites (without maven and cloning the repo to build from source).

Reply:
Let's try to keep the Windows and Linux instruction layout/structure as similar as possible.  Please see https://github.com/dotnet/spark#get-started  .  After your changes are merged, we can pull out the windows instructions into it's own file (similar to yours) and provide links from the main readme to both.